adaptor-rm-webapp/pom.xml

```diff
@@ -58,6 +58,11 @@
         <!-- Specific dependencies -->
         <!-- Start of user code dependencies
         -->
+        <dependency>
```

Comment:
I think this one is no longer necessary

Reply:
Why not? there is always a chance one needs to add own dependencies!

Reply:
I think you just had the same lib required twice?

Reply:
So the comment is about store-core being inserted twice. Ok, it is not important, since the generation willl  not do that.
I thought the comment is about "Start of user code dependencies"